packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -145,7 +147,12 @@ export class Persona {
       reason,
       matchingIdentities
     ): Promise<ReadonlyArray<PublicIdentity>> => {
-      // return all
+      if (isNewSender('http://finnex.com')) {
+        chrome.browserAction.setIcon({ path: 'assets/icons/request128.png' });
+        chrome.browserAction.setBadgeBackgroundColor({ color: [0, 0, 0, 0] });
+        chrome.browserAction.setBadgeText({ text: '*' });
```

Comment:
This will probably fail unit/dom tests due to the missing `chrome` object. Let me show you an approach where this code is only executed in the extension …